File: 1-easy/problems/2496-maximum-value-of-a-string-in-an-array/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def maximumValue(self, strs):
        """
        :type strs: List[str]
        :rtype: int
        """
        return max(int(s) if s.isdigit() else len(s) for s in strs)

File: 1-easy/problems/2496-maximum-value-of-a-string-in-an-array/solution.ts
function maximumValue(strs: any): boolean | number | string | any {
  """
  :type strs: List[str]
  :rtype: int
  """
  return max(int(s) if s.isdigit() else (s).length for s in strs);
}

export { maximumValue };
